pkgs/by-name/ta/tabby-terminal/splice-argv.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/app/lib/cli.ts b/app/lib/cli.ts
+index 03213dd8..2ff31389 100644
+--- a/app/lib/cli.ts
++++ b/app/lib/cli.ts
+@@ -1,9 +1,9 @@
+ import { app } from 'electron'
+ 
+ export function parseArgs (argv: string[], cwd: string): any {
+-    if (argv[0].includes('node')) {
+-        argv = argv.slice(1)
+-    }
++
++    // This is because we are running with electron
```

Comment:
makeWrapper has a thing to set argv0, with that this patch will very likely be unnecessary

Reply:
I already tried setting argv0, but it didn't work. This is probably due to the way electron handles argv when you're launching an app using its CLI.

Reply:
@pluidev also made the same suggestion :rofl: